portal/views/patients.py

```diff
@@ -78,7 +78,8 @@ def render_patients_list(
             patient.current_qb = qb_status['visit_name']
             if research_study_id == EMPRO_RS_ID:
                 patient.clinician = clinician_name_map[patient.clinician_id]
-                patient.action_state = qb_status['action_state']
+                patient.action_state = qb_status['action_state'].title() \
```

Comment:
do we need to add the list of `action_states` to the i18n lookup tables?

Reply:
@pbugni good point, I believe so.  Is this something you can help me with?

Reply:
We extract strings from certain enums with [`get_static_strings()`](https://github.com/uwcirg/truenth-portal/blob/develop/portal/models/i18n.py#L53-L74), but it doesn't look like the `action_states` in `empro_states` can be introspected/enumerated?

Maybe we can address in a followup PR?